pkg/testutils/datapathutils/data_path.go

```diff
@@ -64,7 +65,18 @@ func RewritableDataPath(t testing.TB, relative ...string) string {
 	if !ok {
 		t.Fatal("unable to get caller information")
 	}
-	cockroachWorkspace := filepath.Dir(filepath.Dir(filepath.Dir(thisFilePath)))
+	cockroachWorkspace := thisFilePath
```

Comment:
what is this about? isn't `testutils/datapathutils/data_path.go` always at a fixed depth from the workspace dir?